openpype/hosts/unreal/api/lib.py

```diff
@@ -180,32 +245,32 @@ def create_unreal_project(project_name: str,
     }
 
     if preset["install_unreal_python_engine"]:
-        # If `OPENPYPE_UNREAL_ENGINE_PYTHON_PLUGIN` is set, copy it from there
-        # to support offline installation.
+        # If `PYPE_UNREAL_ENGINE_PYTHON_PLUGIN` is set, copy it from there to
```

Comment:
Why key was changed `OPENPYPE_UNREAL_ENGINE_PYTHON_PLUGIN` -> `PYPE_UNREAL_ENGINE_PYTHON_PLUGIN`?

Reply:
BTW is it used only here? It doesn't seem that path set there is validated.

Reply:
Actually this part is mainly deprecated. I'll mark it so and we'll remove it from future releases.